common/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What we could do is move all the strings to `common` which would mean that you wouldn't need to worry about what strings go in what modules.

If you see in some of my earlier commits, I managed to bring strings to `common` so that they can be used as if they're in the same module by aliasing the strings in `aliases_common.xml`. Just thought it would be nice when new code is written and you don't need to dig around finding the `R.string.sensors` which is in `common.R` file rather than `app.R` or whatever.

The downside with aliasing is that all strings that are moved to `common` would need to be aliased one-by-one.

On the other hand, if all strings were moved to `common`, then no aliasing would be needed.